src/App.test.jsx

```diff
@@ -2,15 +2,31 @@ import React from 'react';
 
 import { render } from '@testing-library/react';
 
+import { useSelector } from 'react-redux';
+
 import App from './App';
 
+jest.mock('react-redux');
+
 test('App', () => {
-  const { getByText } = render((
+  const taskTitle = '';
+  const tasks = [
+    { id: 1, title: '첫번째 할 일' },
+    { id: 2, title: '두번째 할 일' },
+  ];
+
+  useSelector.mockImplementation((selector) => selector({
+    taskTitle,
+    tasks,
+  }));
+
+  const { container, getByText } = render((
     <App />
   ));
 
   expect(getByText(/추가/)).not.toBeNull();
 
-  // TODO: 통합 테스트 코드 작성
-  // CodeceptJS => 실제 브라우저에서 사용자 테스트 실행 가능.
+  tasks.forEach((task) => {
+    expect(container).toHaveTextContent(task.title);
+  });
```

Comment:
구조 분해 할당을 이용해서 이렇게 표현하는 건 어떨까요?

```suggestion
  tasks.forEach(({ title }) => {
    expect(container).toHaveTextContent(title);
  });
```